Tests/MarathonTests/MarathonTests.swift

```diff
@@ -22,7 +22,7 @@ class MarathonTests: XCTestCase {
     }
 
     override func tearDown() {
-        try! folder.delete()
+        try? folder.delete()
```

Comment:
That was changed to avoid crash when using setUp / tearDown from test code. It's not necessary anymore.